app/helpers/world_location_helper.rb

```diff
@@ -1,15 +0,0 @@
-module WorldLocationHelper
-  def group_and_sort(locations)
```

Comment:
looks like it's being called here https://github.com/alphagov/whitehall/blob/main/app/views/world_locations/index.html.erb#L35